packages/core/src/test/shared/errors.test.ts

```diff
@@ -417,3 +418,16 @@ describe('util', function () {
         assert.deepStrictEqual(getErrorMsg(err), 'aws error desc 1')
     })
 })
+
+describe('isNetworkError()', function () {
+    it('returns true when it is a VS Code Proxy error', function () {
```

Comment:
this would fit in the "util" describe() just above here. and can be a single it()